instrumentation/mongo/mongo-common/src/main/java/io/opentelemetry/javaagent/instrumentation/mongo/MongoClientTracer.java

```diff
@@ -80,54 +97,111 @@ protected String dbConnectionString(CommandStartedEvent event) {
     return null;
   }
 
-  @Override
-  public String normalizeQuery(BsonDocument statement) {
-    // scrub the Mongo command so that parameters are removed from the string
-    BsonDocument scrubbed = scrub(statement);
-    return scrubbed.toString();
+  private static final Method IS_TRUNCATED_METHOD;
+
+  static {
+    IS_TRUNCATED_METHOD = Arrays.stream(JsonWriter.class.getMethods())
```

Comment:
This method was added in the 3.7 release of the driver so accessing it reflectively